dev/Common/TerminalVelocityFeatures-AppNotifications.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<features xmlns="http://microsoft.com/windowsappsdk/TerminalVelocity/20210729/TerminalVelocityFeatures.xsd">
+
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. See LICENSE in the project root for license information.
+-->
+
+    <!-- See docs/TerminalVelocity.md for more info.  -->
+
+    <!-- Enabled variants -->
+    <feature>
+        <name>Feature_AppNotifications</name>
+        <description>App Notifications for the WindowsAppRuntime</description>
```

Comment:
Nit: Better description would be "Enables Local App sourced Notifications on the device"

Reply:
done